README.md

```diff
@@ -7,8 +7,8 @@ Please note that the release dates indicated with * below are *estimations*, no
 
 | Branch | Beta Release | Initial Release | Active Support Until | Security Support Until
 |--|--|--|--|--|
-|2.15  | 27 Sep 2019 | 8 Oct 2019 | 8 Oct 2019 | 8 Oct 2019
-|2.16  | 30 Jan 2020* | 13 Feb 2020* | 13 Feb 2020 | June 2020? (TBD)**
+|2.15  | 27 Sep 2019 | 8 Oct 2019 | 13 Feb 2020* | 13 Feb 2020*
+|2.16  | 30 Jan 2020 | 13 Feb 2020* | June 2020? (TBD) | June 2020? (TBD)**
 |3.0   | 27 Jan 2020* | 3 Feb 2020* | TBD | TBD
```

Comment:
As we know we haven't hit the 27th Jan, does it make sense to update with the projected beta/stable release date if we have that determined @alanhartless ?